bigframes/dataframe.py

```diff
@@ -1066,6 +1066,36 @@ def head(self, n: int = 5) -> DataFrame:
     def tail(self, n: int = 5) -> DataFrame:
         return typing.cast(DataFrame, self.iloc[-n:])
 
+    def peek(self, n: int = 5, *, force: bool = True) -> pandas.DataFrame:
```

Comment:
```suggestion
    def peek(self, n: int = 5, *, force: bool = False) -> pandas.DataFrame:
```

Let's do `force=False` as the default for now. If we get complaints, we can reconsider.

Reply:
done, and amended docstring.